### ISSUE TITLE
Flow 0.95.1 with path-to-regexp reports "Cannot call pathToRegexp because a callable signature is missing in module path-to-regexp"

### ISSUE THREAD

**Issue by richsinn** [flow-typed/flow-typed#3274]
Flow version: 0.95.1
path-to-regexp version: 2.0.0

Error message:
```
Cannot call pathToRegexp because a callable signature is missing in module path-to-regexp [1].

 [1]   4│ const pathToRegexp = require('path-to-regexp');
        :
      99│             const regex = pathToRegexp(route.metadata.urlPath, keys);
```

Please let me know what other info you need. FWIW, when I manually modified the flow-typed to be a ["common js" module](https://flow.org/en/docs/libdefs/creation/#toc-declaring-a-commonjs-module) (i.e. using `declare module.exports`), it seems to work. 

Thank you!

**Comment by retyui** [comment]
Thanks!
Fixed in https://github.com/flow-typed/flow-typed/issues/3274

**Comment by richsinn** [comment]
@retyui - much love! 

One thing I noticed about your fix in #3275, it mixes some ESM syntax with CommonJS Syntax. The [flow documentation](https://flow.org/en/docs/libflow documentationdefs/creation/#toc-declaring-a-commonjs-module) (scroll all the way to the bottom) has the following note:

> NOTE: Because a given module cannot be both an ES module and a CommonJS module, it is an error to mix `declare export [...]` with `declare module.exports: ...` in the same `declare module` body.

So for example, we can't mix `declare export type` with `declare module.exports`. FWIW, in my local testing, I think simply removing the `export` keyword for the other declarations should do the trick (though it is beyond my understanding as to why that is case).

**Comment by pascalduez** [comment]
@richsinn I [recently did some testing about this](https://github.com/flow-typed/flow-typed/pull/3267#pullrequestreview-228752374), and the only `export` you can use in a commonjs module declaration is `declare export type`, not `declare export function` for instance. 
Which make sense at the end.
